apps/web/src/views/universalFarms/utils/index.ts

```diff
@@ -8,3 +11,7 @@ export function addQueryToPath(path: string, queryParams: { [key: string]: strin
 
   return `${pathname}?${searchParams.toString()}`
 }
+
+export function getChainFullName(chainId: ChainId) {
+  return CHAINS.find((chain) => chain.id === chainId)?.name
```

Comment:
should use `farmSupporttedChains`, can refactor it later

Reply:
ok